### ISSUE TITLE
Doubt: How to check Apicurio is correctly using the database? 

### ISSUE THREAD

**Issue by SusilRamarao** [Apicurio/apicurio-studio#1243]
Hi guys and @EricWittmann one doubt. I have changed the source db to Postgresql by changing config in datasource and the system-properties part and also initialized with DDL and set "apicurio.hub.storage.jdbc.init" to false. Everything is running fine and apis created are stored and retrieved. But I have a doubt, when I queried the database tables it always shows "0 rows" like data is empty. I thought it is not fetching correctly so i intentionally put the wrong password in xml. I got an error while "Init" showing invalid password and there aren't any other database related Apicurio, when queried with \l.

Looks like everything is fine.. but how to check if it is correctly using the database. Thanks.

Also apart from the documentation, I changed below configuration from:

```xml
<default-bindings context-service="java:jboss/ee/concurrency/context/default" datasource="java:jboss/datasources/ExampleDS"
```
to
```xml
<default-bindings context-service="java:jboss/ee/concurrency/context/default" datasource="java:jboss/datasources/ApicurioDS"
```

because initially I faced an issue, while starting the server i got an error stating that ExampleDS is not intiated.

**Comment by EricWittmann** [comment]
Which database tables did you try to query? Each time you create a new API in Apicurio, a new row will be created in this table:

https://github.com/Apicurio/apicurio-studio/blob/master/back-end/hub-core/src/main/resources/io/apicurio/hub/core/storage/jdbc/hub_postgresql9.ddl#L13

If you are querying that table and it's returning 0 rows, then something is wrong somewhere. :) If that's happening I can't guess what might be happening, but we can narrow it down. I would need to know how you're running it - is it the quickstart, docker, kubernetes, openshift? Any configuration details you can provide (or attach to this issue) would help.

(Note: if you attach any config files please make sure you redact any passwords in them!)

Regarding your change to the default bindings of the javaee subsystem - that is bad.  I have no idea what will happen if you use the Apicurio datasource as the default javaee binding datasource, but I wouldn't do that.

If you can post your config files here I might be able to help - `standalone-apicurio.xml` in particular.

**Comment by SusilRamarao** [comment]
Hi Eric I tried to query "select * from api_designs" and it returned 0 rows. I currently installed using"Getting started" page in an linux instance, 

![image](https://user-images.githubusercontent.com/8603210/90024187-35ad1580-dcd2-11ea-8637-39d4f7d67508.png)

The datasource part

![image](https://user-images.githubusercontent.com/8603210/90024455-80c72880-dcd2-11ea-93ab-0d15baa9e811.png)

I have intentionally put *** for password. Rest all the documentation is same, except the default-binding.

If i change the default-binding to ExampleDS as same as initial configuration, then i get this error

 "WFLYCTL0412: Required services that are not installed:" => ["jboss.naming.context.java.jboss.datasources.ExampleDS"],

![image](https://user-images.githubusercontent.com/8603210/90025580-e23bc700-dcd3-11ea-8f7c-b9753e32d231.png)

**Comment by SusilRamarao** [comment]
In the datasource part should i need to change the <driver> name to the exact jar file name?

**Comment by EricWittmann** [comment]
These settings all look good.  Did you by any chance delete the **ExampleDS** datasource from the `datasources` section?  I'm referring to this:

https://github.com/Apicurio/apicurio-studio/blob/master/distro/quickstart/src/main/resources/standalone/configuration/standalone-apicurio.xml#L165-L172

That's the only reason I can think of why starting would fail with that error.

I think the value of `<driver>` in your datasource sections does need to match the actual name of the driver file that you would drop into the `deployments` directory.  If I recall correctly.

Now, if everything is working you should **definitely** see rows added to that table whenever you create an API design.  If you shut down and restart the application, do you still see the API Designs you created or are they gone?

**Comment by SusilRamarao** [comment]
Hi Eric, ya i removed that part

https://www.apicur.io/studio/docs/switching-to-mysql-or-postgresql

My mistake, actually instead of removing only ApicurioDS part of h2 i also removed the ExampleDS part in h2. Now ExampleDS problem is resolved. But still in DB, when I queried "select * from api_designs" I get 0 rows.

I installed the PostgreSQL like usual  way and I changed the user to postgres and I set the password. I executed DDL, and when I execute "\d", i am able to see apicurio database and I executed "\dt" I am able to see tables. But I am not sure why no rows are there.

I am still able to see the test APIs which i created previously.
